Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Refactored Old Gods Offmap Interface

#### Changelog:
- Recolored the history button on the Old God's offmap screen.
- Changed the diplomatic range button on the Old God's offmap screen.
- Added a Unique Icon for the button that opens the Old God's offmap screens.
- The Old God's Offmap Interface is no longer based on the China Interface, so there should be fewer problems with mods that change China Interface, like "A Bigger Interface"

#### Developer changelog:
- Moved unused `china_dragons` to artist resources.
- Removed duplicates of vanilla .dds files from `gfx/interface/offmap_screen`.
- Renamed a lot of textures. If we ever will make more offmap screens (press x to doubt) with unique graphics, it should help with compatibility.

#### How to test:
- Check that I didn't miss any textures and that the game doesn't crash when you click on any Old God offmap screen button.
